<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>库存主页</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/main_logo.png</normaloff>:/new/prefix1/images/main_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow{background-image: url(:/new/prefix1/images/vi3uyy5q.png);}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>941</width>
      <height>341</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string/>
     </property>
    </column>
    <column>
     <property name="text">
      <string>商品名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>编号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>总数量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>进价</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>售价</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>生产厂家</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>生产日期</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>保质期（月）</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>销售数据</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>修改基础信息</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>374</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>查询商品：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="SearchlineEdit"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="SearchpushButton">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="ReturnpushButton">
       <property name="font">
        <font>
         <family>宋体</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="exitpushButton">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>480</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>966</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>商品信息</string>
    </property>
    <addaction name="actionaddNew"/>
    <addaction name="actiondelete"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>商品管理</string>
    </property>
    <addaction name="actionstockmanage"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>保存</string>
    </property>
    <addaction name="actionsave"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="actionOpen_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="text">
    <string>Save all</string>
   </property>
  </action>
  <action name="actionaddNew">
   <property name="text">
    <string>addNew</string>
   </property>
  </action>
  <action name="actiondelete">
   <property name="text">
    <string>delete</string>
   </property>
  </action>
  <action name="actionstockmanage">
   <property name="text">
    <string>stockmanagement</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
